python/aether_routines.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+from netCDF4 import Dataset
+import datetime as dt
+import numpy as np
+
+def read_aether_header(filelist):
+    r""" Grab ancillary information from the netcdf file
+
+    Get some ancillary information from the netcdf file.  This simply
+    mirrors the gitm header info.
+
+    Parameters
+    ----------
+    filelist: a list of netcdf files. Really, only the last file is used.
+              also, the number of files is recorded.
+
+    Returns
+    -------
+    header: A dictionary containing information about the netcdf file, such
+            as nLons, nLons, nAlts, nVars, variable names, time(s)
+    """
+
+    header = {"nFiles": len(filelist), \
+              "version": 0.1, \
+              "nLons": 0, \
+              "nLats": 0, \
+              "nAlts": 0, \
+              "nVars": 0, \
+              "vars": [], \
+              "time": [], \
+              "filename": [filelist[-1]] }
+    
+    ncfile = Dataset(filelist[-1], 'r')
+    header["nVars"] = 0
+    for var in ncfile.variables.values():
+        s = var.shape
+        if (len(s) == 3):
+            header["nLons"] = s[0]
+            header["nLats"] = s[1]
+            header["nAlts"] = s[2]
+            header["vars"].append(var.name)
+            header["nVars"] += 1
+
+    base = dt.datetime(1965, 1, 1, 0, 0, 0)
+    time = np.array(ncfile.variables['Time'])
+    current = base + dt.timedelta(seconds = time[0])
+    header["time"].append(current)
+            
+    ncfile.close()
+
+    return header
+    
+#-----------------------------------------------------------------------------
+# Read in some variables from one file. Inputs:
+#   - file: netcdf file to read
+#   - vars: list of variable NAMES to read
+# Output:
+#   - data["time"]: datetime of the file
+#   - data[NUMBER]: data that is read in.
+#                   NUMBER does from 0 - number of vars read in (0-3 typical)
+#-----------------------------------------------------------------------------
+
+def read_aether_one_file(file, vars):
+    r""" Read in list of variables from a single netCDF file
+
+    Parameters
+    ----------
+    file: netcdf file to read
+    vars: list of variable NAMES to read
+
+    Returns
+    -------
+    data["time"]: datetime of the file
+    data[NUMBER]: data that is read in.
+                  NUMBER goes from 0 - number of vars read in (0-3 typical)
+
+    """
```

Comment:
```suggestion
    """ Read in list of variables from a single netCDF file

    Parameters
    ----------
    file : str
        Name of netCDF file to read
    vars : list
        List of desired variable names to read

    Returns
    -------
    data : dict
        Dict with keys 'time', which contains a datetime object specifying the time of
        the file and indices ranging from 0 to `len(vars) - 1`, corresponding to the
        variable names in `vars` that holds arrays of the specified dat
    
    """
```

Why is an index used instead of the variable name?  It's an unusual approach.